Resources/Prototypes/Entities/Structures/hydro_tray.yml

```
@@ -102,7 +102,7 @@
     tags:
     - NoPaint
   - type: TTS
-    voice: FactCore # Sunrise-TTS
+    voice: Portal2FactCore # Sunrise-TTS
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)